src/routes/data/handlers.ts

```diff
@@ -117,9 +117,7 @@ const handleRangeRequest = (
   data: ContiguousData,
   dataAttributes: ContiguousDataAttributes | undefined,
 ) => {
-  const ranges = rangeParser(data.size, rangeHeader, {
-    combine: true,
-  });
+  const ranges = rangeParser(data.size, rangeHeader);
```

Comment:
just a thought - this may be something worth noting in the CHANGELOG (i.e. "range requests with adjacent boundaries will no longer be combined and returned as a single stream"). just so we have something to point to if it does introduce unintentional issues with certain HTTP clients